app/src/main/java/com/asfoundation/wallet/promotions/PromotionsFragment.kt

```diff
@@ -60,32 +67,41 @@ class PromotionsFragment : DaggerFragment(), PromotionsView {
 
   override fun setLevelIcons() {
```

Comment:
We should set the name of the methods to reflect that we are dealing with the legacy layout